handlers/error.go

```diff
@@ -307,6 +307,10 @@ func ErrVersionCompare(err error) error {
 	return errors.New(ErrVersionCompareCode, errors.Alert, []string{"failed to compare latest and current version of Meshery"}, []string{err.Error()}, []string{}, []string{})
 }
 
+func ErrGetLatestVersion(err error) error {
+	return errors.New(ErrVersionCompareCode, errors.Alert, []string{"failed to get latest version of Meshery"}, []string{err.Error()}, []string{}, []string{})
```

Comment:
We can have some causes and remedies here :sweat_smile:

Reply:
Ooooh, yes, please. 😄

Reply:
what should I write in the cause, I am just making a request and scrapping the version, I could add "Internet is not connected", or the "repo"/"org" name is wrong, ideas?